### PR TITLE
Highlight the selected vehicle icon

### PR DESCRIPTION
**Pull request by arkadyan**
Asana ticket: [The selected vehicle for should be highlighted on the route ladder](https://app.asana.com/0/1112935048846093/1121935738144311)

Move the selection action to the new triangle icons.

### CODE REVIEW

**Comment by arkadyan** [comment]
Screen Shot 2019-05-15 at 10 04 54

**Comment by ingridpierre** [comment]
looks good, small tweak: can we make the overlay opacity 20% instead of 41 or what it is now? It makes it a little hard to see the state change currently.

File: assets/css/_ladder.scss [hunk 1]
```diff
@@ -21,6 +21,21 @@
...
+  cursor: pointer;
+}
+
+.m-ladder__vehicle-triangle {
+  fill: $blue-grey;
+  stroke: none;
+
+  &.selected {
+    stroke: $dark;
+    stroke-width: 2;
```
Comment (skyqrose) [inline assets/css/_ladder.scss]: We should decide which styling belongs in css and which styling belongs inline when writing the svg. I like this being here. Maybe stroke widths and colors should be in css, and shapes+positioning should be in svg? Would that be good to start standardizing on?

File: assets/src/components/routeLadder.tsx [hunk 2]
```diff
@@ -25,52 +26,49 @@ const Header = ({ route }: { route: Route }) => {
...
-        <br />
-        <em>fraction before:</em>{" "}
-        {vehicle.timepoint_status &&
-          vehicle.timepoint_status.fraction_until_timepoint}
-      </li>
-    </ul>
-  )
-}
+const Vehicle = ({ vehicle }: { vehicle: Vehicle }) => (
+  <ul className="m-route-ladder__vehicle" style={{ border: "1px solid black" }}>
```
Comment (skyqrose) [inline assets/src/components/routeLadder.tsx]: Removing dispatch here will cause merge conflicts with #45 and make it hard/impossible to view overlapped vehicles and vehicles without timepoints (which we're likely to want to view during development). Can we leave it in for now?

File: assets/tests/components/ladder.test.tsx [hunk 3]
```diff
@@ -62,9 +65,116 @@ test("renders a ladder", () => {
...
     .toJSON()
 
   expect(tree).toMatchSnapshot()
 })
+
+test("highlights a selected vehicle", () => {
+  const timepoints = [{ id: "t0" }, { id: "t1" }, { id: "t2" }]
+  const vehicles: Vehicle[] = [
+    {
+      id: "upward",
```
Comment (skyqrose) [inline assets/tests/components/ladder.test.tsx]: I think this test might only need 1 vehicle, which could make the test shorter and easier to read.

File: assets/css/_ladder.scss (hunk 1 above)
Comment (arkadyan) [reply assets/css/_ladder.scss]: Yeah, I think that makes sense. I guess I was sort of conceptualizing it as "put everything in CSS except where you can't", but I think that's in essence the same thing as what you are saying without being as clear.

File: assets/src/components/routeLadder.tsx (hunk 2 above)
Comment (arkadyan) [reply assets/src/components/routeLadder.tsx]: Sure

File: assets/tests/components/ladder.test.tsx (hunk 3 above)
Comment (arkadyan) [reply assets/tests/components/ladder.test.tsx]: I'm going to leave 2 to demonstrate others not being selected.

**Comment by arkadyan** [comment]
@ingridpierre and @skyqrose made all those updates.

Screen Shot 2019-05-15 at 11 46 08

**Pull request merged**